library/src/test/java/bio/terra/landingzone/stairway/flight/StepsDefinitionFactoryTypeTest.java

```diff
@@ -3,6 +3,7 @@
 import static org.junit.jupiter.api.Assertions.assertNotNull;
 import static org.junit.jupiter.api.Assertions.assertNull;
 
+import bio.terra.landingzone.library.landingzones.definition.factories.StepsDefinitionFactoryType;
```

Comment:
Is this import needed? It seems odd that there was no code change related to it.